sequencing/clinical_annotator/Readme.md

```diff
@@ -0,0 +1,12 @@
+<!-- dx-header -->
+# VCF File Annotation (DNAnexus Platform App)
+
+# What does this app do?
+It annotates a VCF File with most recent ClinVar TSV file ( downloaded from ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz ) and with a user provided HGMD PRO vcf file. Because of VCF Normalization difference some annotations will be skipped. It does not function with UCSC chromosome labeling. As a result, the input VCF file(s) will need to have their chr prefix removed.
```

Comment:
"This app annotates a VCF (`*.vcf.gz`) file with the most recent ClinVar TSV(`*.tsv`) file (downloaded from `ftp://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz`) and with a user-provided HGMD PRO VCF (`*.vcf.gz`) file. Because of VCF normalization differences, some annotations will be skipped. The app does **not** function with UCSC chromosome labeling. As a result, the input VCF file(s) cannot include `chr` prefixes."